src/components/__test__/Missions.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Missions from '../missions/Missions';
+import store from '../../redux/store';
+
+describe('Testing mission component', () => {});
+it('should render correctly Mission component', () => {
+  const tree = render(
+    <Provider store={store}>
+      <Missions />
+    </Provider>,
+
+  );
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] You have done an admirable job with the alterations; however, there are still areas that need to be addressed. Your tests are presently failing. Whenever I attempt to remedy one of them, the others become unsuccessful again; please refer to the accompanying video:


https://user-images.githubusercontent.com/69011963/210894617-cc14e00b-cda2-4b9f-a68f-118cbff504c1.mov

Reply:
@Rustamxon7 I am so sorry I forgot to commit my changes to the branch 😆 . It should be alright now.